container_inspect_test.go

```diff
@@ -25,6 +25,8 @@ import (
 )
 
 func TestContainerInspectContainsPortConfig(t *testing.T) {
+	//nerdctl do not support yet ipv6
```

Comment:
This has nothing to with `testutil.DockerIncompatible`. And this test should pass with Docker.